syntaxes/GDScript.tmLanguage.json

```
@@ -61,6 +61,9 @@
 				{ "include": "#operators" },
 				{ "include": "#lambda_declaration" },
 				{ "include": "#class_declaration" },
+				{
+					"include": "#variable_declaration_name"
+				},
```

Follow the existing style:

```suggestion
				{ "include": "#variable_declaration_name" },
```